sdk/orbital/arm-orbital/src/operationsInterfaces/contactProfiles.ts

```diff
@@ -53,51 +55,59 @@ export interface ContactProfiles {
     options?: ContactProfilesGetOptionalParams
   ): Promise<ContactProfilesGetResponse>;
   /**
-   * Creates or updates a contact profile
+   * Creates or updates a contact profile.
    * @param resourceGroupName The name of the resource group. The name is case insensitive.
-   * @param contactProfileName Contact Profile Name
+   * @param contactProfileName Contact Profile name.
    * @param location The geo-location where the resource lives
+   * @param networkConfiguration Network configuration of customer virtual network.
+   * @param links Links of the Contact Profile. Describes RF links, modem processing, and IP endpoints.
```

Comment:
why couldn't codegen generate these parameters?

Reply:
related example parameters in swagger: 
https://github.com/Azure/azure-rest-api-specs/blob/93fbd7d8554d2bb504f85daf4c5b6b87db4abb54/specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/examples/ContactProfileCreate.json#L14
and https://github.com/Azure/azure-rest-api-specs/blob/93fbd7d8554d2bb504f85daf4c5b6b87db4abb54/specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/examples/ContactProfileCreate.json#L27

Reply:
this is the new change in latest tag:
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json#L1646
this parameter is required in latest tag